src/chainstate/stacks/index/marf.rs

```diff
@@ -770,13 +771,24 @@ impl MARF {
         };
         Ok(())
     }
+
+    /// Finish writing the next trie in the MARF.  This persists all changes.
```

Comment:
(Commenting on the comment).  If I'm interpreting the code right, this only finished writing a trie into the MARF that was created in the process of mining a block (as opposed to processing a block)?

Reply:
Yep -- that's right, I'll fix the comment.

Reply:
Fixed in 7523e848a